Origin: Debian dosbox package
Description: define CAN_USE_LOCK in src/ints/bios_keyboard.cpp because Debian's
 libsdl disables a bad behaviour in SDL that treats numlock and scrolllock
 different from all other keys.
Last-Update: 2009-11-30
--- a/src/ints/bios_keyboard.cpp
+++ b/src/ints/bios_keyboard.cpp
@@ -28,7 +28,7 @@
 /* SDL by default treats numlock and scrolllock different from all other keys.
  * Some linux distros disable this bad behaviour. (for example debian) 
  * Define the following if this is the case */
-//#define CAN_USE_LOCK 1
+#define CAN_USE_LOCK 1
 
 static Bitu call_int16,call_irq1,call_irq6;
 
